.github/workflows/job-precommit-python-runners-portability-spark.yml

```diff
@@ -0,0 +1,90 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Test for Python Precommit Runners Portability Spark
+
+
+name: Python Precommit Runners Portability Spark
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */6 * * *'
+  push:
+    branches: ['master', 'release-*']
+    tags: ['v*']
+    paths: [ 'sdks/python/apache_beam/runners/portability/spark_runner.py',
+             'sdks/python/apache_beam/runners/portability/spark_runner_test.py',
+             'sdks/python/apache_beam/runners/portability/spark_uber_jar_job_server.py',
+             'sdks/python/apache_beam/runners/portability/spark_uber_jar_job_server_test.py' ]
```

Comment:
We should run postcommits on all paths, please remove these

Reply:
@damccorm same context, should we remove it? (considering this is a pre-commit job)

Reply:
Hi @damccorm is there any comment on this?